test/ContextTest.js

```diff
@@ -19,6 +19,16 @@ describe('CloudinaryContext', () => {
     expect(img.instance().state.url).to.equal("http://res.cloudinary.com/demo/image/upload/sample");
   });
 
+   it("should render without div", function() {
```

Comment:
@strausr, please also create a test to make sure that not sending isOwnBody or sending false, creates a 'div'